maintainers-circle/README.md

```diff
@@ -1,49 +1,76 @@
+is a time for you to stay connected, learn from and with each other, and 
+discuss topics that are close to you. We'll have a 100 person capacity for each 
+event and depending on demand, may re-run the session. The capacity is set so
+this is not a lecture but 
+
+The only contract of the members of each circle is to abide by the CNCF Code of 
+Conduct, be kind, open as you'd like, and committed to hearing others out. 
+Everyone will have a chance to participate in whatever activity is set forth in 
+that meeting. 
 
 ## Goals:
-Bring CNCF project maintainers🔹 together to:
+Bring CNCF project maintainers🔹together to:
 - Collaborate on strategic issues; tackling size:L OSS challenges together
-- Listening (to build better programs, relationships, and sustainability efforts)
-- Share Best Practices
+- Listen (to build better programs, relationships, and sustainability efforts)
+- Share best practices
 - Participate in peer coaching and seek guidance  
 - Highlight successes
 
-🔹Maintainer Definition  
-A maintainer is anyone recognized as a maintainer or core role of a project by
-that project's standards and governance. You can maintain code and/or docs.
+🔹What's a maintainer? There are a lot of meanings within the community.  
+A maintainer is anyone recognized as a maintainer or in a core role of a CNCF 
+project by
+that project's standards and governance. You can maintain code, docs, policy and
+ more, but you are making decisions.
 Relevant CNCF project examples:  
-- [containerd](https://github.com/containerd/project/blob/master/GOVERNANCE.md#maintainership) lists maintainers and code reviewers as key role
+- [containerd](https://github.com/containerd/project/blob/master/GOVERNANCE.md#maintainership) 
+lists maintainers and code reviewers as key role
 - [Prometheus](https://github.com/prometheus/prometheus/blob/master/MAINTAINERS.md)
 lists maintainers in a MAINTAINERS.md file
-- [Kubernetes](owners link)
-doesn't use the word maintainers but uses an OWNERs file method for ownership by
+- [Kubernetes](https://github.com/kubernetes/community/blob/master/community-membership.md)
+doesn't use the word maintainer but uses an OWNERs file method for ownership by
 listing reviewers and approvers
 
 We recognize the term does not apply across the board perfectly. Maintainers
 is an umbrella succinct term that covers many of your roles and inclusivity.
 
-Important note: CNCF has a pool of listed maintainers for each project that vote on behalf of their project in TOC
-elections. This circle is open to all types of maintainers.
+Important note: CNCF has a pool of listed maintainers for each project that vote
+ on behalf of their project in TOC elections. Some projects have more maintainer 
+ roles than this and the circle welcomes all decision makers. 
 
 
 ## Where
-n candence meetings on Zoom  
+Zoom with break out rooms  
+In person as soon as we can at the next face to face KubeCon tbd  
```

Comment:
I thought we were going to move to weekly meetings and do maintainers circle every other week?